packages/oui-password/README.md

```diff
@@ -62,6 +62,35 @@
 </form>
 ```
 
+#### Trigger Validation
+
+Useful when you have a password field, the validation of which is dependant on another field's value (example: a confirm password field, that is dependant on another password field)
+
+```html:preview
+<form name="form3" novalidate>
+    <oui-field label="Password" size="xl">
+        <oui-password model="$ctrl.modelTriggerValidation">
+            <oui-password-rule validator="$ctrl.checkPasswordLength(modelValue)">
+                Must contain between 8 and 30 characters
+            </oui-password-rule>
+            <oui-password-rule pattern="[0-9]+">
+                Have at least one number
+            </oui-password-rule>
+            <oui-password-rule pattern="[A-Z]+">
+                Have at least capital letter
+            </oui-password-rule>
+        </oui-password>
+    </oui-field>
+    <oui-field label="Confirm Password" size="xl">
+        <oui-password model="$ctrl.modelTriggerValidationConfirmPassword" validation-trigger="$ctrl.modelTriggerValidation">
+            <oui-password-rule validator="$ctrl.checkPasswordMatch(modelValue)">
```

Comment:
I think we could go simpler with a `confirm` or `compare` attribute, since it will be its main use.

```
<oui-password model="$ctrl.model1"></oui-password>
<oui-password model="$ctrl.model2" confirm="$ctrl.model1"></oui-password>
```

The binding could be '=?', and watched if the attribute is present.
This will set `$setValidaty('confirm', value)` when changed and could trigger an error in the `oui-field` (https://github.com/ovh-ux/ovh-ui-angular/blob/master/packages/oui-field/src/field.provider.js#L7).

This seems easier to understand and to use, since the logic is already done and oriented to confirm another password :)

Reply:
This is a really good idea @AxelPeter 
This is  done